src/components/tl/cuda/tl_cuda.h

```diff
@@ -22,6 +22,8 @@
 #endif
 
 #define UCC_TL_CUDA_MAX_PEERS 8
+#define UCC_TL_CUDA_MAX_RINGS 8
```

Comment:
will be any scenario in the future where we want to try mode than 8 rings? instead should we allocate memory dynamically